ansible/roles/postgresql/defaults/main.yml

```diff
@@ -1,4 +1,3 @@
-pg_install: True
```

Comment:
🙌 . one thing to ensure is that this role is idempotent when pg_install is True (i think it is)